CHANGELOG.rst

```diff
@@ -12,6 +12,16 @@ and this project adheres to
 [Unreleased]
 -------------------------------------------------------------------------------
 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Added
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- New option in ``cartridge.config_patch_clusterwide``: ``{never_skip = true}``
```

Comment:
Not sure about naming. Why "never" if it work only for one time. I believe we should have "force" in option name